lib/smb2/client.rb

```diff
@@ -1,7 +1,21 @@
 require 'smb2/packet'
+require 'smb2/tree'
 require 'net/ntlm'
```

Comment:
Please separate requires for gem files from project files so it's easier to track the source like so:

```ruby
#
# Standard Library
#

...

#
# Gems
#

...

#
# Project
#

...
```

If you [remove the unneed project requires](https://github.com/rapid7/smb2/pull/28/files#r28818234), this would just degenerate to

```ruby
#
# Gems
#

require 'net/ntlm'
require 'net/ntlm/client'
```